src/app/(frontend)/[center]/events/events-mobile-filters.tsx

```diff
@@ -1,102 +1,31 @@
-        aria-label="Open filters"
-        className="w-full sm:w-1/2"
-      >
-        <div className="flex items-center gap-2">
-          <span>Filters</span>
-          {hasActiveFilters ? <FilterX width={16} /> : <Filter width={16} />}
-        </div>
-      </Button>
-
-      {isOpen && (
-        <div
-          className="fixed inset-0 bg-black/50 z-40 md:hidden"
-          onClick={() => setIsOpen(false)}
-          aria-hidden="true"
-        />
-      )}
-
-      {/* Drawer */}
-      <div
-        className={`fixed bottom-0 left-0 right-0 z-50 md:hidden bg-white rounded-t-2xl shadow-lg transition-transform duration-300 ease-out h-[90vh] flex flex-col ${
-          isOpen ? 'translate-y-0' : 'translate-y-full'
-        }`}
-      >
-        {/* Handle */}
-        <div className="flex justify-center pt-2 pb-1">
-          <div className="h-1 w-12 bg-gray-300 rounded-full" />
-        </div>
-
-        <div className="flex items-center justify-between px-6 py-4 border-b border-gray-200">
-          <h2 className="text-lg font-semibold">Filters</h2>
-        </div>
-
-        <div className="flex-1 overflow-y-auto px-6">
-          <div className="border-b">
-            <div
-              onClick={() => setTypeOpen(!typeOpen)}
-              className="w-full flex items-center justify-between py-3  transition-colors"
-            >
-              <h3 className="font-semibold">Event Type</h3>
-              <ChevronDown
-                className={`h-5 w-5 transition-transform ${typeOpen ? 'rotate-180' : ''}`}
-              />
-            </div>
-            {typeOpen && (
-              <div className="pb-4">
-                <EventsTypeFilter types={types} />
-              </div>
-            )}
-          </div>
-
-          <div className="border-b">
-            <div
-              onClick={() => setDateOpen(!dateOpen)}
-              className="w-full flex items-center justify-between py-3  transition-colors"
-            >
-              <h3 className="font-semibold">Date Range</h3>
-              <ChevronDown
-                className={`h-5 w-5 transition-transform ${dateOpen ? 'rotate-180' : ''}`}
-              />
-            </div>
-            {dateOpen && (
-              <div className="pb-4">
-                <EventsDatePicker startDate="" endDate="" />
-              </div>
-            )}
-          </div>
-        </div>
-
-        <div className="sticky bottom-0 px-4 py-4 border-t bg-white">
-          <Button variant="callout" onClick={() => setIsOpen(false)} className="w-full px-4 py-3">
-            Show {eventCount} events
-          </Button>
-        </div>
-      </div>
-    </>
+    <MobileFiltersDrawer docLabel="events" docCount={total} hasActiveFilters={hasActiveFilters}>
+      <EventsTypeFilter types={types} />
+      <EventsGroupsFilter groups={groups} />
+      <EventsTagsFilter tags={tags} />
+      <ModesOfTravelFilter />
+      <EventsDatePicker startDate="" endDate="" />
```

Comment:
Let's add a check on these like we have in `blog-mobile-filter`.

```ts
{item.length > 1 && <Filter item={item} />}
```
Probably should put it in the Files themselves since they are reused